From f5d243672ba98ec590b570eb1abdae4eda320fb2 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Wed, 12 Jun 2024 22:13:02 -0700
Subject: [PATCH] Docs/process/email-clients: Document HacKerMaiL

HacKerMaiL (hkml) [1] is a simple tool for mailing lists-based
development workflow, and optimzied for LKML.  It is also actively being
maintained by DAMON maintainer, and recommended for DAMON community.
Add a simple introduction of the tool on the email-clients document.

[1] https://github.com/sjp38/hackermail

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/process/email-clients.rst | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/Documentation/process/email-clients.rst b/Documentation/process/email-clients.rst
index 471e1f93fa09..2c4b5ba83311 100644
--- a/Documentation/process/email-clients.rst
+++ b/Documentation/process/email-clients.rst
@@ -370,3 +370,13 @@ clients may encrypt responses to encrypted mail for all recipients, including
 the mailing lists.
 Unless a way to disable this "feature" is introduced, Proton Mail is unsuited
 to kernel development.
+
+HacKerMaiL (TUI)
+****************
+
+HacKerMaiL (hkml) is a public-inbox based mails management tool that doesn't
+require subscription of mailing lists.  It is developed and maintained by the
+DAMON maintainer to support workflows for DAMON and general kernel subsystems.
+No special installation or configuration other than ``git`` are required.
+Refer to the README (https://github.com/sjp38/hackermail/blob/master/README.md)
+to get started.
-- 
2.39.2

